src/cli/cli-definitions.js

```diff
@@ -174,5 +174,15 @@ export default {
   "verbose": {
     env: "VERBOSE",
     help: "Set the logging to verbose"
+  },
+  "logLevel": {
+    help: "Sets the log level"
+  },
+  "enableConfigChanges": {
+    env: "ENABLE_CONFIG_CHANGES",
```

Comment:
We've been discussing in #1715 what things to expose as env vars, as I think we want to use less env vars in the future. What do you think about making this not be an env var?